README.md

```diff
@@ -20,17 +20,20 @@ De OSPO-NL kennisbank is gepubliceerd op
 
 Voordat lokaal gebouwd kan worden, moeten alle dependencies geïnstalleerd worden:
 
+(gebaseerd op [Python Poetry](https://python-poetry.org/docs/#installation) dependency & package manager)
+
 ```bash
-pip install poetry
 poetry install --no-interaction
 ```
 
 Lokaal kan vervolgens de documentatie site gebouwd en getest worden met (of `run.sh` :wink:):
 
 ```bash
-poetry run mkdocs serve
+poetry run mkdocs serve -a 0.0.0.0:8000
 ```
 
+De site kan dan lokaal benaderd worden op [http://localhost:8000](http://localhost:8000)
```

Comment:
De site kan dan binnen het netwerk benaderd worden.

Reply:
OK .. maar ik denk dat het zo wel voldoende is. Er is nu een gemakkelijke link .. en we kunnen ook niet alle opties hier benoemen. Ik denk dat deze documentatie moet helpen en bijdragen aan het (enigszins gemakkelijk) _kunnen_ draaien en testen .. en eigen kennis en kunde mag toch ook wel verwacht worden van ontwikkelaars? Vind ik tenminste 😊